.github/workflows/test.yml

```diff
@@ -291,7 +302,7 @@ jobs:
 
       - name: Create GCP compute instance
         id: create-instance
-        if: ${{ steps.changed-files-specific.outputs.any_changed == 'true' || github.event.inputs.regenerate-disks == 'true' || github.event_name == 'push'}}
+        if: ${{ needs.build.outputs.any_changed == 'true' || github.event.inputs.regenerate-disks == 'true' }}
```

Comment:
Question: we're repeating this condition twice, is there a way to just depend on the condition from the previous step?

(It's ok if there isn't.)

Reply:
Fixed by [imp(ci): do not repeat conditions if not needed](https://github.com/ZcashFoundation/zebra/pull/4074/commits/3372cd34c863dce304dcf29462dc34d8657e7db1)